gradle.properties

```diff
@@ -2,10 +2,13 @@
 bintrayVersion=1.4
 guavaVersion=16.0.1
 immutablesVersion=2.1.8
-jacksonVersion=2.4.4
+jacksonVersion=2.6.3
```

Comment:
this could mean a backcompat break for folks taking an update. it's probably fine, but we should note this in the release notes.